src/heartbeat/heartbeat.c

```diff
@@ -1,37 +1,83 @@
-//Assume init_uart() and init_can() have been called
+/*
```

Comment:
We can leave this in for now, but in the future I plan on removing all of these headers.

I find that the format doesn't supply much useful information; the filename and dependencies are obvious; the authors, date modified and revision history are all available via git. The notes and description section are useful, but feel out of place.

Also, I find that people tend to litter this section with tabs and spaces, and there are tons of alignments issues, which looks awful.

Reply:
1. Agree filename, dependencies and revision history are redundant. And agree notes and description are out of place (I do feel that too when I edited). 
2. I actually copy the format from PAY code. They have been using this format for quite awhile. Maybe, if you would like, we can set up an universal way on how to document descriptions and notes on our code base?! @vogeldylan @brunofalmeida

Reply:
Actually, I was working on cleaning this up in the pay qual-development branch yesterday. I agree that filename, dependencies, and revision history are redundant.

Reply:
And authors are probably redundant too

Reply:
I would keep the authors. :p Cuz people who contribute to the design, might not be the person committing changes. The purpose of having authors is when there's a question in codes, you know who to ask. And for UTAT, there has been trouble asking people about past work cuz people come and go. So more reference, the safer in my opinions.

Reply:
Alright, that makes sense